### DISCUSSION TITLE
使用GPG防止commiter冒充

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#301] [Announcements]
https://imcbc.cn/202209/gunpg-commiter-sign/

**Comment by penndu** [comment]
都没有注意到，一会我去试试！

**Comment by caibingcheng** [comment]
> @penndu
> 都没有注意到，一会我去试试！

我是工作要求才知道这个东西的，以前完全没关注过😂  （居然可以发表情
